examples/OrmliteExample/app/src/main/java/com/example/androidannotations/ormliteexample/db/DBUtil.java

```diff
@@ -0,0 +1,46 @@
+package com.example.androidannotations.ormliteexample.db;
+
+import android.content.Context;
+
+import com.example.androidannotations.ormliteexample.R;
+
+import org.androidannotations.annotations.EBean;
+import org.androidannotations.annotations.EBean.Scope;
+import org.androidannotations.annotations.RootContext;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.InputStream;
+
+/**
+ * Created by Administrator on 2015/5/1.
+ */
+@EBean(scope = Scope.Singleton)
+public class DBUtil {
+    @RootContext
+    Context context;
+
+    public void initDatabase() {
+        try {
+            InputStream is = context.getResources().openRawResource(R.raw.test);
```

Comment:
I think it is nicer to just create and persist some example data then to create this class.